packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -31,41 +51,221 @@ describe('authentication', () => {
     }
   });
 
-  it('should not allow authentication.authenticate calls before initialization', () => {
-    const authenticationParams: authentication.AuthenticatePopUpParameters = {
-      url: 'https://someurl/',
-      width: 100,
-      height: 200,
-    };
+  describe('Testing authentication.initialize function', () => {
+    it('authentication.initialize should successfully register authentication.authenticate.success/failure handler', async () => {
+      authentication.initialize();
+      const messageForAuthenticationSuccess = utils.findMessageByFunc('authentication.authenticate.success');
+      const messageForAuthenticationFailure = utils.findMessageByFunc('authentication.authenticate.failure');
+      expect(messageForAuthenticationSuccess).toBeNull();
```

Comment:
(I'm asking because I'm genuinely curious, I'm just trying to learn more about this area:) why would we expect the message to be null here? I looked through the files and it seems like we would want that message to have been sent and kept by utils.

Reply:
When we initialize something we don't respond with any function we just tell to initialize it and not send any message.